### DISCUSSION TITLE
Another problem upgrading symfony webpack-encore 0.31.1 to 1.8.2

### DISCUSSION THREAD

**Discussion by neilf184** [symfony/symfony#47775] [Q&A] [answered]
Hi,
I'm attempting to upgrade from symfony webpack-encore 0.31.1 to 1.8.2. After many other problems, I'm now blocked by the following build error (using npm build dev) in modernizr-loader:-

Module build failed (from ./node_modules/modernizr-loader/index.js):
TypeError: this.exec is not a function
    at Object.module.exports (/var/www/dev2.yourdolphin.com/node_modules/modernizr-loader/index.js:24:26)

modernizr v3.12.0
modernizr-loader v1.0.1

Has anyone come across this before or got any suggestions?

**Comment by stof** [comment, marked as answer]
`modernizr-loader` was released 6 years ago for its last release. So my guess is that this custom loader does not support webpack 5.

**Reply by stof** [reply]
And https://github.com/peerigon/modernizr-loader/issues/52 confirms this.

So the solution for you is to stop using this unmaintained loader.

**Reply by neilf184** [reply]
Thanks for the fast reply, most helpful
